src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -85,6 +112,9 @@ typedef bmpl::vector<
     current<UsedParticleCurrentSolver>,
     massRatio<MassRatioElectrons>,
     chargeRatio<ChargeRatioElectrons>
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
```

Comment:
Here, the precompiler statement is necessary because the `HasFlag` condition is based on a complete species list.